ui/ui-showcase.jsx

```diff
@@ -671,10 +671,21 @@
   });
 
   var App = React.createClass({
-
     render: function() {
       return (
         <ShowCase>
+          // Bug 1245666 - select & style or implement react slideshow component
+          <Section name="FTESlideShow">
```

Comment:
This doesn't work at the moment and breaks the showcase, so I suggest dropping it for now.